compiler/rustc_mir_build/src/build/coverageinfo/mcdc.rs

```diff
@@ -150,44 +155,64 @@ impl MCDCState {
         decision_ctx.decision_stack.push_back(lhs);
     }
 
-    fn take_condition(
+    fn try_finish_decision(
         &mut self,
+        span: Span,
         true_marker: BlockMarkerId,
         false_marker: BlockMarkerId,
-    ) -> (Option<ConditionInfo>, Option<MCDCDecisionSpan>) {
```

Comment:
This change looks a bit awkward. It's just a temporary expedient and will be removed in refactored mcdc implementation for pattern matching.